engines/sci/engine/kernel.cpp

```diff
@@ -91,6 +91,20 @@ const Common::String &Kernel::getKernelName(uint number) const {
 	return _kernelNames[number];
 }
 
+Common::String Kernel::getKernelName(uint number, uint subFunction) const {
+	// FIXME: The following check is a temporary workaround for an issue
+	// leading to crashes when using the debugger's backtrace command.
```

Comment:
This comment is copied from the same check in the other `getKernelName` function. It seems to be a very old check from the time we stored kernel functions differently on the stack. I could find no caller of `getKernelName` that should now be able to trigger this, so I've replaced them with asserts in 4b72a42da93da5560ecde38010d328196ff727fd.